### ISSUE TITLE
Vocaliser la validation d'un enfant dans le foyer

### ISSUE THREAD

**Issue by MattiSG** [sgmap/mes-aides-ui#360]
En tant que déficient visuel,
J'entends « Ajouter un nouvel enfant » après en avoir ajouté un,
Pour savoir que l'ajout a été pris en compte

**Comment by fpagnoux** [comment]
![image](https://cloud.githubusercontent.com/assets/11834997/20311972/066f6e90-ab52-11e6-949d-8087c2cf3617.png)

Mettre le focus sur le bouton "Ajouter un enfant" a un effet de bord graphique.

Je ne pense pas qu'il faille le supprimer, on risquerait de réduire la navigabilité en mode clavier. Pour moi c'est acceptable, @MattiSG ?

**Comment by MattiSG** [comment]
Tout à fait acceptable pour moi.
